vscode-powershell.build.ps1

```diff
@@ -101,13 +101,8 @@ task BuildAll BuildEditorServices, Build
 #region Test tasks
 
 task Test Build, {
-    if (!$global:IsLinux) {
```

Comment:
Looks like CI is now failing on Linux

Reply:
So is this more of an issue of CI can't run the tests on Linux? As I mentioned in the commit the only issue I've run into is sometimes I need to `Invoke-Build Clean` because it gets in a weird state thinking there's code that's not actually there.

I'm not seeing anything particularly helpful in the output from the CI either :grimacing:

Reply:
Looks like the tests may work locally but not in a headless CI so let's ignore the tests in CI instead.